pinot-core/src/main/java/org/apache/pinot/core/data/GenericRow.java

```diff
@@ -90,6 +90,11 @@ public boolean isNullValue(String fieldName) {
     return _nullValueFields.contains(fieldName);
   }
 
+  /**
+   * Returns whether this row has null values for any of the columns
+   */
+  public boolean hasNullValues() { return _nullValueFields.size() > 0; }
```

Comment:
(Code style) Expand this into 3 lines.
(nit) `return !_nullValueFields.isEmpty()` seems more readable

Reply:
Done